ansible/roles/operator-pipeline/templates/openshift/tasks/copy-image.yml

```diff
@@ -10,8 +10,12 @@ spec:
       default: "registry.redhat.io/rhel8/skopeo:8.4-13"
     - name: src_image
       description: reference to the source bundle image
-    - name: dest_image
-      description: reference to the destination bundle image
+    - name: dest_image_first_part
```

Comment:
haha :D let's give it a meaningful name. how about dest_image_registry, dest_image_repository, and dest_image_tag?